helpers.json

```diff
@@ -747,6 +747,24 @@
             "canTrickySpringBallJump"
           ]
         },
+        {
+          "name": "h_complexToCarryFlashSuit",
```

Comment:
not sure how i feel about the `To`. It feels a little odd, but i guess i am fine either way.

Reply:
Yeah the idea of the "To" is just to avoid implying that having/carrying a flash suit is a requirement, only that *if* you're carrying one then it is complex to do so